### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Blindspot22**
Improvement on the documentation as demanded by @Hermann-Core and @IngridPuppet

### CODE REVIEW

**Review by IngridPuppet — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -66,24 +66,30 @@ Ensure you have the following installed:
...
+```
 
 ## Example
- 1. Start the mediator service:
+ Start the mediator service:
 
+```sh
 cd didcomm-mediator-rs/mediator
 cargo run
+```
```
[the remark this answers is not in the document]
Comment (Blindspot22) [reply README.md]: Can you check it again if it suits your demand?

File: README.md [hunk 2]
```diff
@@ -66,24 +66,30 @@ Ensure you have the following installed:
 1. Create a working directory eg(cd didcomm-mediator-rs) and cd into your directory.
```
[the remark this answers is not in the document]
Comment (Blindspot22) [reply README.md]: check and let me know

File: README.md [hunk 3]
```diff
@@ -66,24 +66,30 @@ Ensure you have the following installed:
...
 
  2. Clone the repository using the following command:
-
+```sh
 git clone https://github.com/adorsys/didcomm-mediator-rs.git
+```
```
[the remark this answers is not in the document]
Comment (Blindspot22) [reply README.md]: I just added the  SSH alternative

File: README.md [hunk 4]
```diff
@@ -63,27 +63,36 @@ Ensure you have the following installed:
...
-
-cd didcomm-mediator-rs/mediator
-cargo run
-
+ Starting the mediator service:
+ 
+Since the didcomm-mediator-rs is made up of so many `crates` both `libraries` and `binaries`, to run anything you have to explicitly specify the `crate` and the `binary` you want to run like for example
+```sh
+cargo run -p <package> --bin <binary>
+```
```
Comment (IngridPuppet) [inline README.md:92]: Disregard `mediator-server`. The only binary crate is `generic-server`, hence the only crate you can run. You may launch the generic server by selecting some features mapping to the plugins you would want to be loaded. Can you sync with @Christiantyemele please? There is also the important step of configuring env variables that he can support you in documenting.

**Review by IngridPuppet — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Blindspot22) [reply README.md:92]: @Christiantyemele please check this message.

Comment (Blindspot22) [reply README.md:92]: Okay sir

Comment (Christiantyemele) [reply README.md:92]: > Disregard `mediator-server`. The only binary crate is `generic-server`, hence the only crate you can run. You may launch the generic server by selecting some features mapping to the plugins you would want to be loaded. Can you sync with @Christiantyemele please? There is also the important step of configuring env variables that he can support you in documenting.

Ok, :eyes:

Comment (IngridPuppet) [reply README.md:92]: @Blindspot22 - This is no longer blocked. Are you proceeding with it?

Comment (Blindspot22) [reply README.md]: You can check again and let me know if more suggestions!

Comment (IngridPuppet) [reply README.md]: Can you update the branch so we have `readme.png` here?

Comment (Blindspot22) [reply README.md]: Okay sir. Done, I think the image is clear now.

**Review by IngridPuppet — APPROVED** [review]

**Pull request merged**